tests/README.md

```diff
@@ -17,7 +17,18 @@ objects at runtime.
 in the `tests` and `scripts` directories.
 
 To run the tests, follow the [setup instructions](../CONTRIBUTING.md#preparing-the-environment)
-in the `CONTRIBUTING.md` document. In particular, we recommend running with Python 3.9+.
+in the `CONTRIBUTING.md` document. In particular, you have to run with Python 3.9+.
```

Comment:
I changed the wording here because since #9382, `test/utils.py` (which is used by all tests) uses `functools.cache`, which is unavailable in Python 3.8.

Reply:
In practice we've required 3.9+ for a lot longer, due to the use of `ast.unparse` in `tests/check_new_syntax.py` :)

Reply:
Good to know. I believe you could still run other tests, so "recommend" wasn't *too* inaccurate :P

I only noticed because I had 3.8 installed in WSL.